source/read/cursors.txt

```diff
@@ -58,19 +71,40 @@ Retrieve documents from a cursor individually by calling the ``next()`` method.
 
 The following example finds all documents in a collection with a ``name`` value
 of ``"Dunkin' Donuts"``. It then prints the first document in the cursor by calling the
-``next()`` method.
+``next()`` method. Select the :guilabel:`Synchronous` or :guilabel:`Asynchronous` tab to
+see the corresponding code:
+
+.. tabs::
+   
+   .. tab:: Synchronous
+      :tabid: sync
+
+      .. io-code-block::
+         :copyable
+
+         .. input:: /includes/cursors/cursors.py
+            :start-after: start-cursor-next
+            :end-before: end-cursor-next
+            :language: python
+         
+         .. output::
+
+            {'_id': ObjectId('...'), 'address': { ... }, 'borough': 'Bronx', 'cuisine': 'Donuts', 'grades': [...], 'name': "Dunkin' Donuts", 'restaurant_id': '40379573'}
 
-.. io-code-block::
-   :copyable:
+   .. tab:: Asynchronous
+      :tabid: async
 
-   .. input:: /includes/cursors/cursors.py
-      :start-after: start-cursor-next
-      :end-before: end-cursor-next
-      :language: python
+      .. io-code-block::
+         :copyable
 
-   .. output::
+         .. input:: /includes/cursors/cursors-async.py
+            :start-after: start-cursor-next
+            :end-before: end-cursor-next
+            :language: python
+         
+         .. output::
```

Comment:
S: kinda outside the scope of this PR but I'd add visible: false to the io code blocks:
```suggestion
         .. output::
            :visible: false
```

Reply:
applies to all